src/Microdown/MicCodeBlock.class.st

```diff
@@ -116,7 +114,7 @@ MicCodeBlock >> label: aString [
 
 { #category : #accessing }
 MicCodeBlock >> language [
-	^ arguments at: #language
+	^ arguments at: #language ifAbsent: [ '' ]
```

Comment:
No, this should not have an ifAbsent part, that is what lines 68-70 above ensures.